files/brews/dnsmasq.rb

```diff
@@ -1,36 +1,54 @@
-require 'formula'
-
 class Dnsmasq < Formula
-  homepage 'http://www.thekelleys.org.uk/dnsmasq/doc.html'
-  url 'http://www.thekelleys.org.uk/dnsmasq/dnsmasq-2.71.tar.gz'
-  sha1 'b0a39f66557c966629a0ed9282cd87df8f409004'
-  version '2.71-boxen1'
-
-  def options
-    [['--with-idn', "Compile with IDN support"]]
+  desc "Lightweight DNS forwarder and DHCP server"
+  homepage "http://www.thekelleys.org.uk/dnsmasq/doc.html"
+  url "http://www.thekelleys.org.uk/dnsmasq/dnsmasq-2.76.tar.gz"
+  sha256 "777c4762d2fee3738a0380401f2d087b47faa41db2317c60660d69ad10a76c32"
```

Comment:
we need the version to be updated so that we can distinguish between upstream and our version . i.e. `version '2.71-boxen1'`